rs-react-app/src/components/_test_/PokemonDetailsPanel.test.tsx

```diff
@@ -0,0 +1,111 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
fixing react warning about state updates not being included in act(...):
```suggestion
import { act, render, screen, waitFor } from '@testing-library/react';
```
use act or use waitFor